packages/wp-utils/package.json

```diff
@@ -26,5 +26,7 @@
   },
   "type": "module",
   "main": "./src/index.js",
```

Comment:
This should point to `index.ts` now.

Reply:
Other modules do not do this.

Reply:
This package is a bit different from the others since it's private, and not published to npm. That's why it has different fields in `package.json`.

The published packages all point to `dist/index.js`, but here we don't have a `dist` folder and we don't have `index.js` anymore since the file extension is now `.ts`. It makes sense to change this to a file name that exists.

And since this is a private package only really used by webpack I don't think we even need to specify the `source` field.